public/locales/en/flightcontrolpanel.json

```diff
@@ -0,0 +1,30 @@
+{
+  "controller": {
+    "label": "Toggle flight control",
+    "description": {
+      "intro": "Interact with the highlighted area to control the camera.",
+      "mouse-title": "Mouse controls:",
+      "mouse-body": {
+        "text": "Click and drag to rotate. Hold",
+        "list": {
+          "shift": "Shift to zoom (y-axis) or roll (x-axis)",
+          "ctrl": "Ctrl to pan"
+        }
+      },
+      "touch-title": "Touch controls:",
+      "touch-body": {
+        "list": {
+          "1-finger": "1 finger to roate",
+          "2-fingers": "2 fingers to pan",
+          "3-fingers": "3 fingers to zoom (y-axis) or roll (x-axis)"
+        }
+      }
+    }
+  },
+  "settings-title": "Settings",
+  "friction-label": "Friction control",
+  "sensitivity": {
+    "label": "Input sensitivity",
+    "description": "Controls how sensitive the touch and mouse inputs are"
+  }
```

Comment:
Here the naming sceme is also a bit weird. Why do some things live under the "controller" namespace?